cmd/nuntium/mediator.go

```diff
@@ -80,6 +80,7 @@ mediatorLoop:
 				continue
 			}
 			if !mmsEnabled() {
+				log.Print("MMS are disabled")
```

Comment:
`MMS` is not plural so this text doesn't read right to me. It should be `is` instead of `are`. Or perhaps something like `mmsEnabled: disabled` instead.

Reply:
I changed it to "MMS is disabled". The "mmsEnabled: " prefix is for mmsEnabled() function log.